db/db.go

```
@@ -0,0 +1,139 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+	"log"
+
+	sq "github.com/Masterminds/squirrel"
+)
+
+// DB has all the tooling for talking to postgres
+type DB struct {
+	psql sq.StatementBuilderType
+}
+
+// New creates a DB instance
+func New(dbURL string) (*DB, func() error) {
+	postgres, err := sql.Open("pgx", dbURL)
+	if err != nil {
+		log.Fatalln("Unable to connect to the database")
```

It would be nicer for this function to return an error instead of calling `log.Fatalln` inside of a library like this.